src/prepare_problem.py

```diff
@@ -0,0 +1,123 @@
+"""This work was partially funded by the EuroHPC PL project,
+funded in the frame of Smart Growth Operational Programme, topic 4.2.
+
+Authors: Mateusz Hurbol, Justyna Zawalska
+"""
+
+import inspect
+import math
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import Dict, List, Tuple
+
+import networkx as nx
+import pandas as pd
+import wfcommons
+from dataclasses_json import dataclass_json
+
+from src.utils.file_management import read_json_file
```

Comment:
```suggestion
from .utils.file_management import read_json_file
```
I don't think that `src` is needed here